### PR TITLE
Added support for BTCEXP_BASEURL

### PR DESCRIPTION
**Pull request by ketominer**
Should close https://github.com/janoside/btc-rpc-explorer/issues/205

### CODE REVIEW

**Comment by Kixunil** [comment]
Does it require rewrite rule in nginx?

File: views/layout.pug [hunk 1]
```diff
@@ -2,24 +2,25 @@ doctype html
 html(lang="en")
 	head
 		meta(charset="utf-8")
+		base(href="/btc-rpc-explorer/")
```
Comment (Kixunil) [inline views/layout.pug:5]: I believe you intended to use variable here. It breaks when e.g. `/btc-explorer/` is used

**Comment by ketominer** [comment]
> Does it require rewrite rule in nginx?
> 
> Edit: looks like it does.

sample nginx config:
  location ^~ /btc-rpc-explorer {
    proxy_pass http://btc-rpc-explorer;
  }

with BTCEXP_BASEURL=/btc-rpc-explorer/

**Comment by ketominer** [comment]
BTW I noticed I missed one place (link to bitcoin fun in top menu)

File: views/layout.pug (hunk 1 above)
Comment (ketominer) [reply views/layout.pug:5]: good find!

**Comment by Kixunil** [comment]
Hmm, it works for me even without trailing slash when I use rewrite (not sure why, didn't check the code so deeply to understand it that well).

**Pull request merged**